README.md

```diff
@@ -72,23 +88,61 @@ vars:
     lever_requisition_passthrough_columns: ['the', 'list', 'of', 'fields']
 ```
 
+### Change the build schema
+By default, this package builds the Lever staging models within a schema titled (`<target_schema>` + `_stg_lever`) and your Lever modeling models within a schema titled (`<target_schema>` + `_lever`) in your destination. If this is not where you would like your Lever data to be written to, add the following configuration to your root `dbt_project.yml` file:
+
+```yml
+models:
+    lever_source:
+      +schema: my_new_schema_name # leave blank for just the target_schema
+    lever:
+      +schema: my_new_schema_name # leave blank for just the target_schema
+```
+    
+### Change the source table references
+If an individual source table has a different name than the package expects, add the table name as it appears in your destination to the respective variable:
+
+> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_lever/blob/main/dbt_project.yml) variable declarations to see the expected names.
```

Comment:
This way it is easier to see what the source tables are directly from the source package.
```suggestion
> IMPORTANT: See this project's [`dbt_project.yml`](https://github.com/fivetran/dbt_lever_source/blob/main/dbt_project.yml) variable declarations to see the expected names.
```